### PR TITLE
Change route part

### PR DESCRIPTION
**Pull request by Mazorius**
I duplicate route and change it to routes.

Reason for that is the following error:
`Loading configuration file failed" file=/opt/alertmanager/config/alertmanager.yml err="cannot have continue in root route"`

### CODE REVIEW

**Comment by Mazorius** [comment]
Can add the DCO afterwards?

Last time I add something I do not need this. Sorry :(

**Comment by SuperQ** [comment]
Yes, if you `git commit -s --amend` then `git push -f` it should add the DCO.

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -130,6 +130,31 @@ current node.
...
+# How long to wait before sending a notification again if it has already
+# been sent successfully for an alert. (Usually ~3h or more).
+[ repeat_interval: <duration> | default = 4h ]
+
+# Zero or more child routes.
+routes:
+  [ - <routes> ... ]
+```
+
+## `<routes>`
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: This is not the syntax of a routes, it's the syntax of a route

Comment (Mazorius) [reply content/docs/alerting/configuration.md]: I added route and delete stuff from routes. I think the diff is not correctly shown.

**Comment by gouthamve** [comment]
@brian-brazil Whats the latest on this?

**Comment by brian-brazil** [comment]
The existing documentation is correct as far as I can see, this can be closed.

**Pull request closed without merging**